### PR TITLE
Add "scherbelberg"

### PR DESCRIPTION
**Pull request by s-m-e**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by s-m-e** [comment]
@conda-forge/help-python This package is ready for review.

File: recipes/scherbelberg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scherbelberg-{{ version }}.tar.gz
+  sha256: 8ff1665653b60110549896bf808dc963b972641af8c21ca46650eb0353d0a03c
+
+build:
+  number: 0
+  noarch: python
```
Comment (BastianZim) [inline recipes/scherbelberg/meta.yaml:14]: Upstream uses `scripts` so this cannot be no arch anymore: https://github.com/pleiszenburg/scherbelberg/blob/ee3615984ab3ca797e19ba0434c20b92f3aff611/setup.py#L105

Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:14]: @BastianZim I am also the upstream maintainer: Shall I simply remove the line `scripts=[],`? I am actually using `entry_points` ... `scripts` is probably a relic from an old project template.

Comment (BastianZim) [reply recipes/scherbelberg/meta.yaml:14]: Ahh great, that makes it much simpler! Yes, that would be the best approach. 

But we’d need a new version with that too or you can apply it as a patch here - whatever you find easier.

Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:14]: I'll push a new release to PyPI and update this very PR. Should take 10 minutes plus CI checks ... I'll ping you when it's done.

Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:14]: @BastianZim Done. CI test done, too - ready for another review.

File: recipes/scherbelberg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - "scherbelberg = scherbelberg._cli:cli"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+    - click
+    - dask
```
Comment (BastianZim) [inline recipes/scherbelberg/meta.yaml:26]: Are you sure that you need the full `dask` package or just `core`? See: https://docs.dask.org/en/latest/install.html#conda

File: recipes/scherbelberg/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% set name = "scherbelberg" %}
```
Comment (BastianZim) [inline recipes/scherbelberg/meta.yaml:1]: Love the name by the way. :)

File: recipes/scherbelberg/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    - hcloud
+    - pyyaml
+    - typeguard
+
+test:
+  imports:
+    - scherbelberg
+    - scherbelberg._cli
+  commands:
+    - pip check
```
Comment (BastianZim) [inline recipes/scherbelberg/meta.yaml:36]:
```suggestion
    - pip check
    - scherbelberg --help
```
Just to check the CLI.

File: recipes/scherbelberg/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scherbelberg-{{ version }}.tar.gz
+  sha256: a87fb41eb666a3a5d5b0c1aa2caaa2e10a646c382bb88a4373c997dd1dc2a708
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - "scherbelberg = scherbelberg._cli:cli"
```
Comment (BastianZim) [inline recipes/scherbelberg/meta.yaml:17]:
```suggestion
    - scherbelberg = scherbelberg._cli:cli
```
Just cosmetics. Or was that added deliberately?

File: recipes/scherbelberg/meta.yaml (hunk 2 above)
Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:26]: This is intentional

File: recipes/scherbelberg/meta.yaml (hunk 3 above)
Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:1]: Thanks :) Have you read the [project's name section](https://github.com/pleiszenburg/scherbelberg#projects-name) in the readme?

File: recipes/scherbelberg/meta.yaml (hunk 4 above)
Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:36]: Good idea.

File: recipes/scherbelberg/meta.yaml (hunk 5 above)
Comment (s-m-e) [reply recipes/scherbelberg/meta.yaml:17]: No, just stolen from another conda-forge package that I used as a template.

**Comment by s-m-e** [comment]
Ping @BastianZim - applied requested changes, CI passed. Ready for yet another review.

File: recipes/scherbelberg/meta.yaml (hunk 5 above)
Comment (BastianZim) [reply recipes/scherbelberg/meta.yaml]: Ahh ok. Just for future reference, if the package is python, the easiest way to generate the recipe is using [Grayskull](https://github.com/conda-incubator/grayskull). It's not always perfect but pretty close to it.

**Comment by BastianZim** [comment]
Looks great – thanks!

@conda-forge/staged-recipes Should be ready for merging.

**Comment by ocefpaf** [comment]
Great review and fast upstream release. This PR is a packager's dream!

**Comment by s-m-e** [comment]
@ocefpaf Glad that the PR was a packager's dream ;) The merge happened about 7 hours ago but I have not yet seen a matching feedstock repository being created. From past experience, this usually only takes a few minutes. Did something go wrong or do I simply need to be patient a little longer?

**Comment by BastianZim** [comment]
Seems like a problem with Travis: https://github.com/conda-forge/staged-recipes/runs/5058311531?check_suite_focus=true#step:6:965

@conda-forge/core Could this be related to the ongoing Travis situation?

**Comment by s-m-e** [comment]
Issue resolved, feedstock showed up.

**Pull request merged**